tests/unit/lms/assets_test.py

```diff
@@ -0,0 +1,97 @@
+import os
+from unittest import mock
+
+import pytest
+
+from lms.assets import _CachedFile
+
+
+class Test_CachedFile:
```

Comment:
These tests have been added:

a) Because there weren't any
b) Because I modified the file and dropped the coverage failing the build